src/object/C4Movement.cpp

```diff
@@ -136,17 +159,25 @@ void C4Object::StopAndContact(C4Real & ctco, C4Real limit, C4Real & speed, int32
 int32_t C4Object::ContactCheck(int32_t iAtX, int32_t iAtY, uint32_t *border_hack_contacts, bool collide_halfvehic)
 {
 	// Check shape contact at given position
-	Shape.ContactCheck(iAtX,iAtY,border_hack_contacts,collide_halfvehic);
+	Shape.ContactCheck(iAtX, iAtY, border_hack_contacts, collide_halfvehic);
 
 	// Store shape contact values in object t_contact
-	t_contact=Shape.ContactCNAT;
+	t_contact = Shape.ContactCNAT;
 
 	// Contact script call for the first contacted cnat
 	if (Shape.ContactCNAT)
-		for (int32_t ccnat=0; ccnat<4; ccnat++) // Left, right, top bottom
+	{
+		for (int32_t ccnat = 0; ccnat < 4; ccnat++) // Left, right, top bottom
+		{
 			if (Shape.ContactCNAT & (1<<ccnat))
```

Comment:
Since you're adding whitespace everywhere else, the shifts here should get some too

Reply:
Ok